patterns/molecules/navigation/primary-nav.js

```diff
@@ -1,20 +0,0 @@
-/**
```

Comment:
this file is being called in organisms/header/header.js. I think this header.js can be deleted

Reply:
deleted